### PR TITLE
ACM- 13528-Application-list-'System'-can-vary

### PR DESCRIPTION
**Pull request by jeswanke**
Signed-off-by: John Swanke <jswanke@redhat.com>

### CODE REVIEW

**Comment by KevinFCormier** [comment]
@jeswanke I tested this out and it works well, but it looks like we should keep `open-cluster-management` in the list of prefixes and add the RHACM installation namespace as well, because some components have a static installation namespace.
image
image

File: backend/src/routes/aggregators/applicationsOCP.ts [hunk 1]
```diff
@@ -265,12 +266,19 @@ function getValues(labels: { annotation: any; value: any }[]) {
   }
 }
 
+export const systemAppNamespacePrefixes: string[] = []
+export async function discoverSystemAppNamespacePrefixes() {
+  if (!systemAppNamespacePrefixes.length) {
+    systemAppNamespacePrefixes.push('openshift')
+    systemAppNamespacePrefixes.push('hive')
+    systemAppNamespacePrefixes.push('rhacm')
+    systemAppNamespacePrefixes.push('open-cluster-management')
```
Comment (KevinFCormier) [inline backend/src/routes/aggregators/applicationsOCP.ts:275]: Sorry John - by "RHACM installation namespace" I still meant you should look up what that actually is... not the hardcoded `rhacm` namespace that I used as an example.
```suggestion
    systemAppNamespacePrefixes.push('open-cluster-management')
    const mch = await getMultiClusterHub()
    if (mch?.metadata?.namespace && mch.metadata.namespace !== 'open-cluster-management') {
      systemAppNamespacePrefixes.push(mch.metadata.namespace )
    }
```

Comment (jeswanke) [reply backend/src/routes/aggregators/applicationsOCP.ts]: I got to remember to see if there's a suggested change 🥇

**Pull request merged**